paper-ieee-trans.tex

```diff
@@ -669,7 +669,7 @@ \section{Conclusion}
 delivery of packages. Thereby, safe and frequent software releases are doable, reducing
 the impact on production systems. In this regard, a pre-production validation,
 through the preview testbeds or resource centers early adoption -- handled by the
-Staged--rollout process -- alleviates the likelihood of discovering uncovered
+Staged--rollout process, alleviates the likelihood of discovering uncovered
```

Comment:
this change breaks the "-- whatever --" structure, not needed

Reply:
ok

Reply:
ok